src/pages/Wishlist/wishList.jsx

```diff
@@ -0,0 +1,140 @@
+import { useEffect, useReducer, useState } from "react/cjs/react.development";
+import { useWishList } from "../../context/WishlistContext";
+import { MakiCross11 } from "../../assets/icons/icons.jsx";
+import "./wishList.css";
+import axios from "axios";
+import { useCart } from "../../context/CartContext";
+import { cartFn } from "../Cart/cartFn";
+
+export function Wishlist() {
+  const { wishList, setWishList } = useWishList();
+  const { cart, setCart } = useCart();
+  const [setBtnDisable] = useState(false);
+  const [dispatch] = useReducer(cartFn, {
+    items: [],
+    qty: 1,
+    subTotal: 0,
+  });
+  const removeHandler = (productid) => {
+    (async () => {
+      try {
+        const token = localStorage.getItem("login");
+        const res = await axios({
+          method: "DELETE",
+          url: `/api/user/wishlist/${productid}`,
+          headers: { authorization: token },
+        });
+        setWishList(res.data.wishlist);
+        console.log("response from delete", res.data.wishlist);
+      } catch (error) {
+        console.error(error);
+      }
+    })();
+  };
+
+  useEffect(() => {
+    const token = localStorage.getItem("login");
+
```

Comment:
Suggestion : Add these functionality to context , to make it more resuable and wherever else you want to call